.github/workflows/release_snapshot.yml

```diff
@@ -0,0 +1,41 @@
+# **what?**
+# This workflow will....
```

Comment:
Can we get these comments filled out so we can verify the Action is doing what it is intended to do?

Reply:
filled them out!